ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -466,10 +467,17 @@ ColumnLayout {
 
             onClicked: root.collectibleClicked(model.chainId, model.contractAddress, model.tokenId, model.symbol, model.tokenType)
             onRightClicked: {
+                let ownedByUser = false
+                const ownedByAccount = ModelUtils.get(model.ownership, 0)
+                if (!!ownedByAccount && !!ownedByAccount.accountAddress) {
+                    ownedByUser = ModelUtils.contains(root.ownedAccountsModel, "address", ownedByAccount.accountAddress, Qt.CaseInsensitive)
+                }
```

Comment:
> I'm afraid that it's even more complicated because probably we may have more than one entry in `ownership` model for `ERC1155` tokens. Then inspecting only the first one may be not sufficient.

🙈 I'll update! Should'd be too costly to check all accounts agains all owners assuming there's not much data in both models.

I'll have a quick look in `nim` to check if we can easily do it. Otherwise I'll create a task to take the time needed for it.

Reply:
I'll go with the QML impl and add a task to update the nim model..

Reply:
done!